src/services/authService.js

```diff
@@ -115,7 +115,7 @@ export default class AuthService {
     const payload = Helper.pickFields(user, ['id', 'roleId']);
     const token = await Helper.genToken(payload);
 
-    return { token, ...Helper.omitFields(user, ['password']) };
+    return { token, ...Helper.omitFields(user, ['password', 'createdat', 'updatedat']) };
```

Comment:
I thought @SamuelKoroh fixed this {'createdat', 'updatedat'}, why do we still have it?